### PR TITLE
Add OpenSMOG to conda packages

### PR DESCRIPTION
**Pull request by junioreif**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by junioreif** [comment]
@conda-forge-admin, please ping team

File: recipes/OpenSMOG/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cbeda5f8dcadf8be64ffe6959d3102ea92c93eed14c5f2a1efc15ec76960ca96
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - CLINAME=OpenSMOG._cli:main
```
Comment (xhochy) [inline recipes/OpenSMOG/meta.yaml:17]: Is this really the name of the entry point?

Comment (junioreif) [reply recipes/OpenSMOG/meta.yaml:17]: yeah, I used to use it this way, there is something weird?

Comment (xhochy) [reply recipes/OpenSMOG/meta.yaml:17]: CLINAME is a weird/undescriptive name for an executable in global scope. I would have expected this to be more like `opensmog=...`. If this is how it is currently setup, we can merge.

**Pull request merged**